contracts/DAIInterestRateModelV4.sol

```diff
@@ -0,0 +1,132 @@
+pragma solidity ^0.8.10;
+
+import "./JumpRateModelV2.sol";
+
+/**
+  * @title Compound's DAIInterestRateModel Contract (version 4)
+  * @author Compound, Dharma (modified by Maker Growth)
+  * @notice The change from v3 to v4 of this contract was just the `SECONDS_PER_BLOCK` constant,
+  * as noted in https://github.com/compound-finance/compound-protocol/issues/230
+  */
+contract DAIInterestRateModelV4 is JumpRateModelV2 {
+    uint256 private constant BASE = 1e18;
+    uint256 private constant RAY_BASE = 1e27;
+    uint256 private constant RAY_TO_BASE_SCALE = 1e9;
+    uint256 private constant SECONDS_PER_BLOCK = 12;
+
+    /**
+     * @notice The additional margin per block separating the base borrow rate from the roof.
+     */
+    uint public gapPerBlock;
+
+    /**
+     * @notice The assumed (1 - reserve factor) used to calculate the minimum borrow rate (reserve factor = 0.05)
+     */
+    uint public constant assumedOneMinusReserveFactorMantissa = 0.95e18;
```

Comment:
The current reserve factor on the DAI market is 15%, so it should be this:
```suggestion
    uint public constant assumedOneMinusReserveFactorMantissa = 0.85e18;
```

Reply:
makes sense! 👍